department-of-veterans-affairs/va.gov-cms
# VACMS-3248: Hide time end on all timezone field instances.

## Description
See #3248

## Testing done
Visual

## Screenshots
![image](https://user-images.githubusercontent.com/2404547/97599594-8d5c4280-19de-11eb-8245-c4d7f541fe5d.png)

## QA steps

- As an administrator, go to `admin/structure/types/manage/event/form-display` and enable field_datetime_range_timezone on form.
- Go to `/node/9511/edit` and visually verify that `All day` checkbox is no longer visible.
